pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstanceRunnable.java

```diff
@@ -150,9 +161,9 @@ JavaInstance setupJavaInstance() throws Exception {
         // start the state table
         setupStateTable();
         // start the output producer
-        setupOutput(typeArgs[1]);
+        setupOutput();
```

Comment:
Looks like we are never using the typeArgs?

Reply:
yup we can remove